draft-meynell-panrg-scion-research-questions.md

```diff
@@ -457,40 +423,40 @@ However, we have identified a number of possible issues (not comprehensive):
 Critically, the SCION header needs to contain the SRC address as seen by the border router so that the border router can forward incoming response packets to the correct NAT device and port.
 
 Possible solutions:
 
-* With IPv6 underlay, this problem disappears. // TODO Clarify why it disappears? IS the idea that we can remove NATs if everyone would use IPv6?
+* With IPv6 underlay, this problem disappears. // TODO Clarify why it disappears? Is the idea that we can remove NATs if everyone would use IPv6?
 * Introduce a mechanism so that the SCION border router can report the NATed address to an endpoint (similar to a STUN server).
 
 # Dataplane stability
 
 ## Link load balancing
 
-Links may get overloaded because the SCION routing system fails to distribute load properly over different links. New/different links might be underutilized.
+Links may get overloaded because the SCION distributed path selection process fails to distribute load properly over different links, resulting in uneven utilization.
 
-If links become overloaded, there are several ways to handle that. Non comprehensively:
+There are several potential approaches to relieve an overloaded link:
 
-* Squeeze: send an SCMP message to trigger end-hosts to use an alternative path
-* Steer: send and SCMP to trigger users to ask control server for a better path
-* Reduce: hand over very short lived paths, let the end-hosts wait for the path to expire so that they request new paths and (hopefully) decide on a different path.
-* Recommend: let the end-hosts know which paths are recommended by the AS at this time.
+* introduce explicit congestion notifications from routers to endpoints.
+* optimize the path lookup process so that the control plane hands out paths that optimize load balancing.
 
-If a link has good properties, many AS will disseminate segments, therefore paths that go through this link and the link may become overloaded. See Simon Scherrer's work on Braess Paradox.
+<!--
+If a link has good properties, many ASes will disseminate segments and therefore paths through this link to the extent link may become overloaded. See Simon Scherrer's work on Braess Paradox.
 
-Either there needs to be some constant control by all clients to not choose the best theoretical path, but the one that works best. Or we need to find a way that control servers do not disseminate “good” links to all end-hosts.
+Either there needs to be some constant control by all clients to not choose the best theoretical path, but the one that works best, or there needs to be a mechanism whereby control plane do not disseminate “good” links to all end-hosts.
 
 The current consensus is that end-hosts can use multi-pathing and “automatically” converge on the best path, i.e. creating an equilibrium. Again, see Simon Scherrer's work on Braess Paradox.
-
+-->
```

Comment:
This text has some issues. I'd therefore comment it and eventually leave ti for a future iteration..
* I think the first 4 bullet points can be summarized by my bullet points above
* there are forward reference to Simon Scherer's work 
* what is `The current consensus`? Among who? Why?